pkg/server/backend_manager.go

```diff
@@ -194,6 +194,7 @@ func NewDefaultBackendManager() *DefaultBackendManager {
 
 // NewDefaultBackendStorage returns a DefaultBackendStorage
 func NewDefaultBackendStorage(idTypes []pkgagent.IdentifierType) *DefaultBackendStorage {
+	metrics.Metrics.SetBackendCount(0)
```

Comment:
nit: add a comment for why this is being done.

Reply:
Done.

/lgtm